docs/articles_en/documentation/openvino_ir/operation_sets/operations_specifications/detection/Proposal_4.rst

```diff
@@ -145,7 +145,7 @@ the second optional tensor of shape ``[batch_size * post_nms_topn]`` with probab
   * **Description**: *framework* specifies how the box coordinates are calculated.
   * **Range of values**:
 
-    * "" (empty string) - calculate box coordinates like in Caffe
+    * "" (empty string) - calculate box coordinates
```

Comment:
same... This relates indirectly.

Reply:
i agree with @kblaszczak-intel, let's leave this reference

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/dcc29c2247b8ff64a8d80be7591eff02ce87d58f